### PR TITLE
Fix drag-and-drop on Meeting Updates

### LINKED ISSUE #1627: Error while dragging cards between columns during update phase
## Issue 

See: https://sentry.io/parabol/action-staging/issues/430712459/

About to take off on a plane, so being brief.

Repro:

1. Put multiple cards in one column (all same owner)
2. Put only one column next door (same owner)
3. Start meeting, proceed to updates
4. Drag card from column with multiple cards to column containing only one

- **OS:** macOS
- **Browser:** Chrome
ONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by dan-f**
Fixes #1627

### CODE REVIEW

File: src/universal/modules/teamDashboard/containers/TeamColumns/TeamColumnsContainer.js [hunk 1]
```diff
@@ -30,9 +31,7 @@ class TeamColumnsContainer extends Component {
   }
 
   getProjectById = (projectId) =>
-    this.props.viewer.projects.edges
-      .map(({ node }) => node)
-      .find(({ id }) => projectId === id);
+    getProjectById(this.props.viewer.projects, projectId);
```
Comment (mattkrick) [inline src/universal/modules/teamDashboard/containers/TeamColumns/TeamColumnsContainer.js]: this.getProjectById calling getProjectById is confusing, but i'm cool to ship now fix later

Comment (dan-f) [reply src/universal/modules/teamDashboard/containers/TeamColumns/TeamColumnsContainer.js]: Just emulating currying.

Comment (mattkrick) [reply src/universal/modules/teamDashboard/containers/TeamColumns/TeamColumnsContainer.js]: maybe this is a good place for some of that newage currying all the kids are doing ;-)

Comment (dan-f) [reply src/universal/modules/teamDashboard/containers/TeamColumns/TeamColumnsContainer.js]: Well, we either write the function as explicitly curried, or we mess with `Function.prototype` to make everything automatically curried...

**Comment by dan-f** [comment]
So I merged this into epic8.5 because I did not know that epic8.5 had already merged to master, and that there was additional epic8.5 work on the master branch. I'm going to redo this PR against master. Then we'll go to staging.

**Comment by dan-f** [comment]
So I merged this into epic8.5 because I did not know that epic8.5 had already merged to master, and that there was additional epic8.5 work on the master branch. I'm going to redo this PR against master. Then we'll go to staging.

**Pull request merged**